subsys/net/lib/lwm2m/lwm2m_engine.c

```diff
@@ -3217,6 +3261,131 @@ int lwm2m_perform_read_op(struct lwm2m_message *msg, uint16_t content_format)
 	return ret;
 }
 
+int lwm2m_perform_composite_read_op(struct lwm2m_message *msg, uint16_t content_format, struct lwm2m_obj_path path_list[], uint8_t path_list_size)
+{
+	struct lwm2m_engine_obj_inst *obj_inst = NULL;
+	struct lwm2m_engine_res *res = NULL;
+	struct lwm2m_engine_obj_field *obj_field;
+	int ret = 0, index;
+	uint8_t num_read = 0U;
+
+	/* set output content-format */
+	ret = coap_append_option_int(msg->out.out_cpkt,
+				     COAP_OPTION_CONTENT_FORMAT,
+				     content_format);
+	if (ret < 0) {
+		LOG_ERR("Error setting response content-format: %d", ret);
+		return ret;
+	}
+
+	ret = coap_packet_append_payload_marker(msg->out.out_cpkt);
+	if (ret < 0) {
+		LOG_ERR("Error appending payload marker: %d", ret);
+		return ret;
+	}
+
+	/* Add object start mark */
+	engine_put_begin(&msg->out, &msg->path);
+
+	//Read resouse by path
```

Comment:
"Read resource from path"

Also, use `/*  */` instead of `//` to follow a common style.

Reply:
Ok I will change to /* */ format.